tycho-its/src/test/java/org/eclipse/tycho/test/surefire/JUnit5Test.java

```diff
@@ -75,10 +79,22 @@ public void testJUnit56Runner() throws Exception {
 		verifier.executeGoal("verify");
 		verifier.verifyErrorFreeLog();
 		String projectBasedir = verifier.getBasedir();
-		assertTestMethodWasSuccessfullyExecuted(projectBasedir, "bundle.test.JUnit56Test",
-				"myFirstJUnit56Test(TestInfo)");
+		assertTestMethodWasSuccessfullyExecuted(projectBasedir, "bundle.test.JUnit56Test", "My 1st JUnit 5.6 test!");
 		// make sure test tagged as 'slow' was skipped
-		assertNumberOfSuccessfulTests(projectBasedir, "bundle.test.JUnit56Test", 1);
+		assertNumberOfSuccessfulTests(projectBasedir, "bundle.test.JUnit56Test", 4);
+	}
+
+	@Test
+	public void testJUnit5Reporting() throws Exception {
+		var verifier = getVerifier("/surefire.junit56/bundle.test", false);
```

Comment:
For now I've modified the existing `/surefire.junit56/bundle.test` but might be cleaner with its own project to avoid changing the existing test?